### PR TITLE
Fix breaking change with latest zap version

### PR DESCRIPTION
**Pull request by theisenmark**
**Description**
The staging deploy for `reservations` is currently broken after upgrading to zap v1.14.1

Stack trace of the failed pod:
```
{"level":"info","ts":1585052719.3260384,"logger":"jaeger","caller":"tracing/tracing.go:83","msg":"jaeger tracer configured","git_sha":"ae36c5","version":"0.11.1","enabled":false}
{"level":"info","ts":1585052719.4172933,"caller":"reservation/api.go:50","msg":"using production monolith client","git_sha":"ae36c5","version":"0.11.1","monolithBaseURI":"http://monolith-spothero-django.staging.local:80"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xac1ae2]

goroutine 1 [running]:
github.com/spothero/tools/sentry.(*Core).Enabled(0xc000202e40, 0xc00046c0ff, 0x203000)
	<autogenerated>:1 +0x32
go.uber.org/zap/zapcore.multiCore.Enabled(0xc000208d40, 0x2, 0x2, 0x4062ff, 0xb73480)
	/go/pkg/mod/go.uber.org/zap@v1.14.1/zapcore/tee.go:53 +0x5e
go.uber.org/zap.(*Logger).check(0xc0002008a0, 0xff, 0xc000627830, 0x2e, 0xc00062d5c0)
	/go/pkg/mod/go.uber.org/zap@v1.14.1/logger.go:263 +0x923
go.uber.org/zap.(*Logger).Debug(0xc0002008a0, 0xc000627830, 0x2e, 0xc00062d5c0, 0x1, 0x1)
	/go/pkg/mod/go.uber.org/zap@v1.14.1/logger.go:178 +0x44
github.com/spothero/tools/http.NewMetrics(0xdc2f40, 0xc000144960, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/spothero/tools@v0.34.0/http/metrics.go:131 +0xc57
github.com/spothero/reservation/pkg/reservation.HandlerConfig.NewHandler(0xc0000340be, 0x30, 0xc0003b15e0, 0xc0001a5720)
	/src/pkg/reservation/api.go:55 +0x1c3
main.main.func1()
	/src/cmd/cmd.go:35 +0x77
main.main.func3(0xc8e274, 0xb, 0xc0000380f8, 0x7, 0xda137c, 0x6, 0xdc28c0, 0x28, 0x0, 0x0, ...)
	/src/cmd/cmd.go:47 +0x2e
github.com/spothero/tools/service.Config.ServerCmd.func1(0xc000240000, 0x136c078, 0x0, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/spothero/tools@v0.34.0/service/service.go:192 +0x9ce
github.com/spf13/cobra.(*Command).execute(0xc000240000, 0xc0001161a0, 0x0, 0x0, 0xc000240000, 0xc0001161a0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0xc000240000, 0xc000204500, 0xc99305, 0x11)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
	/src/cmd/cmd.go:54 +0x30b
```

This PR should fix the null pointer error by specifically setting a `LevelEnabler` for the Sentry logging Core.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -317,13 +319,22 @@ github.com/yudai/golcs v0.0.0-20170316035057-ecda9a501e82/go.mod h1:lgjkn3NuSvDf
...
+go.uber.org/atomic v1.6.0 h1:Ezj3JGmsOnG1MoRWQkPBsKLe9DwWD9QeXzTRzzldNVk=
+go.uber.org/atomic v1.6.0/go.mod h1:sABNBOSYdrvTF6hTgEIbc7YasKWGhgEQZyfxyTvoXHQ=
 go.uber.org/multierr v1.1.0 h1:HoEmRHQPVSqub6w2z2d2EOVs2fjyFRGyofhKuyDq0QI=
 go.uber.org/multierr v1.1.0/go.mod h1:wR5kodmAFQ0UK8QlbwjlSNy0Z68gJhDJUG5sjR94q/0=
+go.uber.org/multierr v1.5.0 h1:KCa4XfM8CWFCpxXRGok+Q0SS/0XBhMDbHHGABQLvD2A=
+go.uber.org/multierr v1.5.0/go.mod h1:FeouvMocqHpRaaGuG9EjoKcStLC43Zu/fmqdUMPcKYU=
+go.uber.org/tools v0.0.0-20190618225709-2cfd321de3ee h1:0mgffUl7nfd+FpvXMVz4IDEaUSmT1ysygQC7qYo7sG4=
+go.uber.org/tools v0.0.0-20190618225709-2cfd321de3ee/go.mod h1:vJERXedbb3MVM5f9Ejo0C68/HhF8uaILCdgjnY+goOA=
 go.uber.org/zap v1.10.0 h1:ORx85nbTijNz8ljznvCMR1ZBIPKFn3jQrag10X2AsuM=
 go.uber.org/zap v1.10.0/go.mod h1:vwi/ZaCAaUcBkycHslxD9B2zi4UTXhF60s6SWpuDF0Q=
```
Comment (briancharous) [inline go.sum]: Run `go mod tidy` to clean up this file

Comment (benjigoldberg) [reply go.sum]: I ran this and no changes were found

**Comment by benjigoldberg** [comment]
@theisenmark I'm merging this to unblock work on reservation. Thanks for fixing this.

/merge

**Pull request merged**